client/components/clonePost/index.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-underscore-dangle */
+/* global React, crNonce */
+
+const {
+  data: {
+    select,
+  },
+  i18n: {
+    __,
+  },
+} = wp;
+
+/**
+ * Slotfill to add clone & replace controls to the sidebar.
+ */
+const ClonePost = () => {
+  const currentPost = select('core/editor').getCurrentPost();
+
+  /**
+   * We only are interested in draft posts here.
+   */
+  if (currentPost.status === 'draft') {
+    return null;
+  }
+
+  return (
+    <div>
+      <a href={`/wp-admin/admin-post.php?action=clone_post&p=${currentPost.id}&_wpnonce=${crNonce}`}>
```

Comment:
We should have a way to get the wp-admin root URL from WP itself rather than hardcoding it here. On a multisite with subdirectories, or WP installed in a subdirectory generally, the path will be different. I did a quick search and I don't _think_ there is anything in Gutenberg to do this natively, so we may need to `wp_localize_script` with the admin script that includes this functionality so that we can pass the result of `admin_url()` from PHP to JS.